files/1.14/kubelet.service

```diff
@@ -5,7 +5,7 @@ After=docker.service
 Requires=docker.service
 
 [Service]
-ExecStartPre=/sbin/iptables -P FORWARD ACCEPT
+ExecStartPre=/sbin/iptables -P FORWARD ACCEPT -w 5
```

Comment:
Can you remove this change? I don't see any documented reason for this

Reply:
Hey @micahhausler 

This was actually already implemented in:

https://github.com/awslabs/amazon-eks-ami/commit/128a7149b6b50f0392d9a6951e24bd30f19f8e2a

I added it to my fork for the time being as it helped an issue we had

Reply:
We should wait for it to complete, so 👍for this change.

Reply:
thanks @mogren  , made sense to me to do the same for 1.14
cheers :)